### PR TITLE
PIM-4896: remove alert in js

### PR DESCRIPTION
**Pull request by solivier**
| Q                  | A
| ------------------ | ---
| Migration script?  | n
| Behats             | y
| Specs & Static     | y
| Changelog updated  | n

### CODE REVIEW

**Comment by jjanvier** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/field.js [hunk 1]
```diff
@@ -174,8 +175,7 @@ define([
              */
             setValues: function (values) {
                 if (values.length === 0) {
-                    /*global alert: true */
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/field.js]: you should rename it for console not at the beginning of the file :)

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/field.js [hunk 2]
```diff
@@ -174,8 +175,7 @@ define([
              */
             setValues: function (values) {
                 if (values.length === 0) {
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/field.js]: you should also test if values isset

**Comment by jmleroux** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/field.js [hunk 3]
```diff
@@ -173,9 +173,9 @@ define([
...
             setValues: function (values) {
-                if (values.length === 0) {
-                    /*global alert: true */
-                    alert('value array is empty');
+                if (_.isUndefined(values) && values.length === 0) {
+                    /*global console: true */
+                    console.log('value array is empty');
                 }
 
                 this.model.set('values', values);
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/field.js]: what happens if `values` is undefined ?

**Comment by jmleroux** [comment]
GTM

**Pull request merged**